### ISSUE TITLE
API Key Security Schemes with whitespace

### ISSUE THREAD

**Issue by calthejuggler** [PaloAltoNetworks/docusaurus-openapi-docs#350]
### Is your feature request related to a problem?

Our users will need to input their API Key in the form of "Api-Key aRandomApiKey", but I cannot seem to have whitespace in the Authorization input.

### Describe the solution you'd like

Whitespace as allowed in the Authorization input.

### Describe alternatives you've considered

Adding a regex pattern (`^Api-Key\s\S+`) to the `scheme` field in the securitySchemes component, but I believe that isn't what that field is for 😂. Same with adding a `schema` and `pattern` field, but I don't think that's allowed.

### Additional context

The security parameter is global and looks like:
```
security:
  - ApiKeyAuth: []
```

It's component looks like:
```
components:
  securitySchemes:
    ApiKeyAuth:
      type: apiKey
      in: header
      description: A User API Key for API authorization.
      name: Authorization
```

Thank you in advance for any thoughts, solutions or fixes!

**Comment by sserrata** [comment]
Hi @calthejuggler, I believe this behavior occurs because we are currently using `trim()` to remove leading/trailing whitespace from input. That said, we can remove this additional step to allow users to include spaces in their keys/tokens/passwords.

**Comment by sserrata** [comment]
Also, just to clarify, you're header value should ultimately be something like:

```
-H 'Authorization: Api-Key 1234567890'
```

Correct?

**Comment by calthejuggler** [comment]
@sserrata That's it!

**Comment by sserrata** [comment]
This functionality is available to preview in canary release `0.0.0-521`. I appreciate if you could test and provide feedback. Thanks!
